### PR TITLE
#306: Updating to reflect need to use wp-config.php instead of the da…

### PR DESCRIPTION
**Pull request by matt-wittmann**
Updating WordPress documentation for the need for a user to go into wp-config.php to set their custom domain instead of through the dashboard. Also noting they will need to update their DNS records.

### CODE REVIEW

File: WordPress/wordpress-custom-domain-configuration.md [hunk 1]
```diff
@@ -6,9 +6,24 @@
...
+## Prerequisite
+
+You have added a CNAME, ALIAS, or similar DNS record between your custom domain and your supplied WordPress site
+domain. For example, you could create a CNAME record like this:
+
+```
+www.example.com.    CNAME   example.useast.wordpress.ctl.io.
+```
+
 ## WordPress Custom Domain Configuration
```
Comment (jim-phillips) [inline WordPress/wordpress-custom-domain-configuration.md]: Please change ## headings to ###.

File: WordPress/wordpress-custom-domain-configuration.md [hunk 2]
```diff
@@ -39,24 +54,24 @@ CenturyLink Cloud WordPress supports custom domain name configuration after [set
...
+8. Look for the lines:
 
-  ![](../images/wp_custom_domain_configuration/wp_custom_domain_configuration_09.png)
+   ```
+   define('WP_SITEURL', getenv("SITEURL"));
+   define('WP_HOME', getenv("SITEURL"));
+   ```
 
-10. Scroll to the bottom and click Save Changes
+9. Replace both instances of `getenv("SITEURL)` with a URL using your new domain. For example:
```
Comment (jim-phillips) [inline WordPress/wordpress-custom-domain-configuration.md]: missing second " in 'getenv("SITEURL)'

**Comment by matt-wittmann** [comment]
jim-phillips : Done.

File: WordPress/wordpress-custom-domain-configuration.md (hunk 1 above)
Comment (osbornm) [inline WordPress/wordpress-custom-domain-configuration.md]: @matt-wittmann can you change the `##` to `###` per @jim-phillips

**Comment by osbornm** [comment]
@matt-wittmann did you see the commit about changing `##` to `###`?

**Comment by matt-wittmann** [comment]
@osbornm, @jim-phillips: Done.

**Pull request merged**